### ISSUE TITLE
Template error when viewing clients in v2.2.3

### ISSUE THREAD

**Issue by Jibhat** [GhostManager/Ghostwriter#208]
**Describe the bug**

**To Reproduce**

1. Build and run docker containers following guide here: https://www.ghostwriter.wiki/getting-started/installation
2. Create new client, filling in all available inputs
3. Try to save client
4. Error produced on attempted redirect to new client view (/rolodex/clients/1)

**Expected Behavior**
Client appears to be saved correctly, and can be used to create projects etc. However it is not possible to view the details as this error results every time.

**Screenshots**
![image](https://user-images.githubusercontent.com/7405666/162908539-8e1d945d-17a7-4d4f-8b4b-f27dedd8a519.png)

**Server Specs:**
 - OS: Ubuntu 20.04 (AWS)
 - Docker version 20.10.7, build 20.10.7-0ubuntu5~20.04.2
 - docker-compose version 1.25.0, build unknown
 - Ghostwriter v2.2.3-rc2, released Feb 2022

**Additional context**
Traceback below:

Template error:
In template /app/ghostwriter/templates/base_generic.html, error at line 71
   clean() got an unexpected keyword argument 'styles'
```
   61 :     <body>
   62 :         <div class="wrapper">
   63 :             <!-- Sidebar Holder -->
   64 :             {% if user.is_authenticated %}
   65 :                 <nav id="sidebar" class="{% if request.session.sidebar.sticky %}active{% endif %}">
   66 :                     <!-- Sidebar Top Section -->
   67 :                     <div class="sidebar-header col d-flex align-items-center justify-content-center">
   68 :                         <a href="{% url 'home:dashboard' %}">
   69 :                             <img src="{% static 'images/all_white_header.png' %}" alt="Ghostwriter">
   70 :                         </a>
   71 :                          <div  class="sidebar-header-tab">
   72 :                             <div class="square">
   73 :                                 <i class="sidebar-toggle-btn fas fa-angle-double-right {% if request.session.sidebar.sticky %}fa-flip-horizontal{% endif %}"></i>
   74 :                             </div>
   75 :                         </div>
   76 :                     </div>
   77 : 
   78 :                     <div class="sidebar-body">
   79 :                         <!-- Project Shortcut Section -->
   80 :                         <ul class="list-unstyled components">
   81 :                             <p class="sideheader icon icon home-icon">Project Shortcuts</p>

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/asgiref/sync.py", line 451, in thread_handler
    raise exc_info[1]
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 38, in inner
    response = await get_response(request)
  File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 263, in _get_response_async
    response = await sync_to_async(response.render, thread_sensitive=True)()
  File "/usr/local/lib/python3.8/site-packages/asgiref/sync.py", line 414, in __call__
    ret = await asyncio.wait_for(future, timeout=None)
  File "/usr/local/lib/python3.8/asyncio/tasks.py", line 455, in wait_for
    return await fut
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.8/site-packages/asgiref/sync.py", line 455, in thread_handler
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/template/response.py", line 105, in render
    self.content = self.rendered_content
  File "/usr/local/lib/python3.8/site-packages/django/template/response.py", line 83, in rendered_content
    return template.render(context, self._request)
  File "/usr/local/lib/python3.8/site-packages/django/template/backends/django.py", line 61, in render
    return self.template.render(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 170, in render
    return self._render(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 162, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/loader_tags.py", line 150, in render
    return compiled_parent._render(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 162, in _render
    return self.nodelist.render(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/loader_tags.py", line 62, in render
    result = block.nodelist.render(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 312, in render
    return nodelist.render(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 938, in render
    bit = node.render_annotated(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 211, in render
    nodelist.append(node.render_annotated(context))
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 905, in render_annotated
    return self.render(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 988, in render
    output = self.filter_expression.resolve(context)
  File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 698, in resolve
    new_obj = func(obj, *arg_vals)
  File "/usr/local/lib/python3.8/site-packages/django_bleach/templatetags/bleach_tags.py", line 23, in bleach_value
    bleached_value = bleach.clean(value, **args)

Exception Type: TypeError at /rolodex/clients/1
Exception Value: clean() got an unexpected keyword argument 'styles'

**Comment by Jibhat** [comment]
Editing ./requirements/base.txt and updating django-bleach resolved the issue

**Comment by chrismaddalena** [comment]
@Jibhat Thanks for reporting this and using the template! There was an update to `bleach` that broke compatibility with `django-bleach`. I pinned `bleach` to the version prior to April 7th. It looks like `django-bleach` released an update on Friday to address this so now updating resolves it. That's a bit easier.

**Comment by LMarina7** [comment]
Hi, I just followed the instructions in [https://www.ghostwriter.wiki/getting-started/installation](https://www.ghostwriter.wiki/getting-started/installation) and I am still getting this error. I create a new client and it does create this client, but when I go to click on it, I get the same error above at http://localhost:8000/rolodex/clients/1. 

I've tried updating django-bleach, but still getting the same error. Any guidance on how to remedy this?

**Comment by Lstesting7** [comment]
Hi, I just followed the instructions in [https://www.ghostwriter.wiki/getting-started/installation](https://www.ghostwriter.wiki/getting-started/installation) and I am still getting this error. I create a new client and it does create this client, but when I go to click on it, I get the same error above at http://localhost:8000/rolodex/clients/1.

I've tried updating django-bleach, but still getting the same error. Any guidance on how to remedy this?
